server/db.go

```diff
@@ -395,6 +395,53 @@ func (c *Server) fetchIssuersByCohort(issuerType string, issuerCohort int16) (*[
 	return &issuers, nil
 }
 
+func (c *Server) fetchIssuerByType(ctx context.Context, issuerType string) (*Issuer, error) {
+	if c.caches != nil {
+		if cached, found := c.caches["issuer"].Get(issuerType); found {
+			// TODO: check this
+			return cached.(*Issuer), nil
+		}
+	}
+
+	var issuerV3 issuer
+	err := c.db.GetContext(ctx, &issuerV3,
+		`SELECT *
```

Comment:
might be good to explicitly say what we are selecting for the issuer.  sometimes if we perform a db migration on deploy the old containers will freak out because select * might no longer fit the Issuer type in the prior running containers.

realize we are doing select * everywhere in this repo tho, can be deferred in a new issue to correct.

Reply:
https://github.com/brave-intl/challenge-bypass-server/issues/276